server/routes/items.js

```diff
@@ -223,12 +230,13 @@ router.put(
     const balanceResponse = await plaid.accountsBalanceGet(balanceRequest);
 
     const account = balanceResponse.data.accounts[0];
-    const latestAccount = await updateBalances(
+    await updateBalances(
       accountId,
       account.balances.current,
       account.balances.available
     );
-    res.json(latestAccount);
+    const latestAccount = await retrieveAccountsByItemId(itemId);
```

Comment:
did `updateBalances` not give the `latestAccount`? wondering why we need to double up on API calls for this